common/model/iiif.js

```diff
@@ -6,7 +6,7 @@ export type IIIFImageServiceSize = {|
 
 export type IIIFImageService = {|
   '@id': string,
-  sizes: IIIFImageServiceSize[],
+  sizes?: IIIFImageServiceSize[],
```

Comment:
When is this nullable?

Reply:
The thumbnail service has sizes, but the regular image service (that we use for the main image) doesn't.

Reply:
Is it worth typing them out as two separate things?